exasol_script_languages_container_ci/lib/branch_config.py

```diff
@@ -0,0 +1,49 @@
+from enum import Enum, auto
+import re
+
+
+class BuildSteps(Enum):
+    BUILD_ALL_ALWAYS = auto()
+    TEST_ALWAYS = auto()
+    REBUILD = auto()
+    PUSH_TO_DOCKER_RELEASE_REPO = auto()
+
+
+class BranchConfig(Enum):
+    DEVELOP = {BuildSteps.BUILD_ALL_ALWAYS: True, BuildSteps.REBUILD: True, BuildSteps.TEST_ALWAYS: True,
+               BuildSteps.PUSH_TO_DOCKER_RELEASE_REPO: False}
+    MASTER = {BuildSteps.BUILD_ALL_ALWAYS: True, BuildSteps.REBUILD: True, BuildSteps.TEST_ALWAYS: True,
```

Comment:
We probably should call this main and use the term master only in regex for backward compatibility